hal/halx86/smp/mps/mps.c

```diff
@@ -9,14 +9,18 @@
 
 #include <hal.h>
 #include <smp.h>
+
 #define NDEBUG
 #include <debug.h>
 
 /* GLOBALS ********************************************************************/
 
-PROCESSOR_IDENTITY HalpStaticProcessorIdentity[MAXIMUM_PROCESSORS] = {{0}};
-PPROCESSOR_IDENTITY HalpProcessorIdentity = NULL;
-UINT32 PhysicalProcessorCount = 0;
+static ULONG PhysicalProcessorCount;
+
+#if 0 // Unused (yet)
```

Comment:
Why comment this out? Since it's referenced in the initializer below, it shouldn't result in any warnings.

Reply:
Initially, the reference was commented out.
Now, (re-)enabled, though actually "unused" yet.